pkg/unit_tests/runtime.go

```diff
@@ -0,0 +1,123 @@
+package unit_tests
+
+import (
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/rancher/k3s/pkg/daemons/config"
+	"github.com/rancher/k3s/pkg/daemons/control/deps"
+	"github.com/rancher/k3s/pkg/version"
+)
+
+// All certs and other files are stored in /tmp/k3s/<RANDOM_STRING>/
+// for testing
+func GenerateTestDataDir(cnf *config.Control) error {
+	var err error
```

Comment:
This can be removed and an colon added on line 17.

Reply:
Removed any  "var err error" and just changed the first time to "err :="